.github/workflows/go-test-lint.yml

```
@@ -0,0 +1,37 @@
+      - go.mod
+      - go.sum
+
+jobs:
+  # Тестирование
+  go-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Установка Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)